lib/src/Rating/SVGComponent.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState, useEffect } from "react";
+
+export type SVGProps = JSX.IntrinsicElements["div"] & {
+    colors?: [string, string];
+    width: React.ReactText;
+    height: React.ReactText;
+    title?: string;
+    index?: number;
+    value?: number;
+    step?: number;
+    customSVG?: any;
+};
```

Comment:
Why don't you just use type `svg` instead of `div`? You don't really have a `div` to spread these props on.
```suggestion
export type SVGProps = JSX.IntrinsicElements["svg"] & {
    colors?: [string, string];
    index?: number;
    value?: number;
    step?: number;
    customSVG?: any;
};
```

Reply:
btw, you don't need `width`, `height`, and `title` since it's already part of `JSX.IntrinsicElements["svg"]`

Reply:
title is not apparently, so using name instead